content/id/docs/concepts/scheduling/scheduling-framework.md

```diff
@@ -0,0 +1,249 @@
+---
+title: Kerangka Kerja Penjadwalan (Scheduling Framework)
+content_template: templates/concept
+weight: 60
+---
+
+{{% capture overview %}}
+
+{{< feature-state for_k8s_version="1.15" state="alpha" >}}
+
+Kerangka kerja penjadwalan (_Scheduling Framework_) adalah arsitektur yang dapat 
+dipasang (_pluggable_) pada penjadwal Kubernetes untuk membuat kustomisasi 
+penjadwal lebih mudah. Hal itu dilakukan dengan menambahkan satu kumpulan "plugin" 
+API ke penjadwal yang telah ada. _Plugin_ dikompilasi ke dalam penjadwal. 
+Beberapa API memungkinkan sebagian besar fitur penjadwalan diimplementasikan 
+sebagai _plugin_, sambil tetap mempertahankan penjadwalan "inti" sederhana dan 
+terpelihara. Silahkan merujuk pada [proposal desain dari kerangka penjadwalan]
+[kep] untuk informasi teknis lebih lanjut tentang desain kerangka kerja 
+tersebut.
+
+[kep]: https://github.com/kubernetes/enhancements/blob/master/keps/sig-scheduling/20180409-scheduling-framework.md
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+# Alur kerja kerangka kerja
+
+Kerangka kerja penjadwalan mendefinisikan beberapa titik ekstensi. _Plugin_ penjadwal
+mendaftar untuk dipanggil di satu atau lebih titik ekstensi. Beberapa _plugin_ ini
+dapat mengubah keputusan penjadwalan dan beberapa hanya bersifat informasi.
+
+Setiap upaya untuk menjadwalkan satu Pod dibagi menjadi dua fase, ** Siklus Penjadwalan (_Scheduling Cycle_) ** dan ** Siklus Pengikatan (_Binding Cycle_) **.
```

Comment:
![image](https://user-images.githubusercontent.com/2609951/79129312-33dc4f80-7dcf-11ea-88ee-3fefbc3e6b51.png)

rendered as plain text, and should be bold for both _fase_